src/main/java/com/exasol/errorcodecrawlermavenplugin/config/ErrorCodeConfigReader.java

```diff
@@ -26,7 +26,7 @@ public ErrorCodeConfigReader(final Path projectDir) throws ErrorCodeConfigExcept
         this.errorConfigFile = projectDir.resolve(CONFIG_NAME).toFile();
         if (!this.errorConfigFile.exists()) {
             throw new ErrorCodeConfigException(messageBuilder("E-ECM-9")
-                    .message("Could not find errorCodeConfig.yml in the current project.")
+                    .message("Could not find " + CONFIG_NAME + " in the current project.")
```

Comment:
Does it work like this? I always used a parameter for it :o

Reply:
yes. But only for constants. For variables you have to use parameters